Frontend/ui-customer/src/Components/MessagingView/View.purs

```diff
@@ -0,0 +1,515 @@
+chatHeaderView config push =
+  linearLayout
+  [ orientation VERTICAL
+  , height $ WRAP_CONTENT
+  , width MATCH_PARENT
+  , clickable true
+  , id $ getNewIDWithTag "MessagingHeaderView"
+  , accessibility DISABLE
+  ][ linearLayout
+    [ width MATCH_PARENT
+    , height WRAP_CONTENT
+    , gravity CENTER_VERTICAL
+    , orientation HORIZONTAL
+    , margin $ MarginTop 12
+    , accessibility DISABLE
+    , padding $ PaddingHorizontal 8 16
+    ][ linearLayout
+        [ height $ V 40
+        , width $ V 40
+        , gravity CENTER
+        , accessibilityHint "Back : Button : Return to ride details"
+        , accessibility ENABLE
+        , onClick push $ const BackPressed
+        ][ imageView
+          [ imageWithFallback $ fetchImage FF_COMMON_ASSET "ny_ic_chevron_left"
+          , height $ V 24
+          , width $ V 24
+          , accessibility DISABLE
+          ]
+        ]
+      , headerNameView config push
+      , headerActionView config push
+      ]
+    , linearLayout
+      [ width MATCH_PARENT
+      , height $ V 1
+      , margin $ MarginTop 12
+      , background Color.grey900
+      ][]
+  ]
+
+headerNameView :: forall w. Config -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+headerNameView config push =
+  let lang = (getValueToLocalStore LANGUAGE_KEY)
+  in
+  linearLayout
+  [ height WRAP_CONTENT
+  , width (V (((screenWidth unit)/10)* 6 ))
+  , orientation VERTICAL
+  ][linearLayout
+    [ height WRAP_CONTENT
+    , width WRAP_CONTENT
+    , accessibilityHint $ "Chat With : " <> config.userConfig.userName
+    , accessibility ENABLE
+    ][ textView $
+      [ text $ if lang == "HI_IN" then config.userConfig.userName <> " " else getString CHAT_WITH <> " "
+      , color Color.black800
+      , ellipsize true
+      , singleLine true
+      , margin $ MarginBottom 8
+      , accessibility DISABLE
+      ] <> if lang == "HI_IN" then FontStyle.body15 TypoGraphy else FontStyle.tags TypoGraphy
+    , textView $
+      [ text $ if lang == "HI_IN" then getString CHAT_WITH else config.userConfig.userName
+      , color Color.black800
+      , ellipsize true
+      , singleLine true
+      , accessibility DISABLE
+      , margin $ MarginBottom 8
+      ] <> if lang == "HI_IN" then FontStyle.tags TypoGraphy else FontStyle.body15 TypoGraphy
+    ]
+   , horizontalScrollView
+     [ height WRAP_CONTENT
+     , width MATCH_PARENT
+     , scrollBarX false
+     , accessibility DISABLE
+     , disableKeyboardAvoidance true
+     ][ linearLayout
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , accessibility DISABLE
+        ][ linearLayout
+            [ height WRAP_CONTENT
+            , width WRAP_CONTENT
+            , cornerRadius 4.0
+            , margin $ MarginRight 6
+            , stroke $ "1,"<> Color.black900
+            , background Color.yellow900
+            , accessibilityHint $ "Vehicle Number : " <>  (STR.replaceAll (STR.Pattern "") (STR.Replacement " ")  config.vehicleNo )
```

Comment:
Define this as function in where clause and pass `config.vehicleNo`/ `config.otp` to it

```
splitString :: String -> String
splitString str = (STR.replaceAll (STR.Pattern "") (STR.Replacement " ")  str ) 
```